lib/core/presentation/pages/ai/ai_page.dart

```diff
@@ -51,6 +59,7 @@ class AIPageState extends State<AIPage> {
   final keyboardVisibilityController = KeyboardVisibilityController();
   Timer? _timer;
   bool _needScrollToEnd = false;
+  bool _selectedCommand = false;
```

Comment:
should rename, where `selectedCommand` sounds like noun, the bool value should be adjective or adverb, 
recommend => `commandSelected`